benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -477,7 +477,7 @@ int main(int argc, char *argv[])
     else if (node == "cuda")
     {
 #ifdef KOKKOS_ENABLE_CUDA
-      using Node = Kokkos::Cuda;
+      using Node = Kokkos::Device<Kokkos::Cuda, Kokkos::CudaSpace>;
```

Comment:
This change does not belong to this PR.  At the very least should be its own commit.

Reply:
It has its own commit now.